validation/json-validator/src/main/java/com/redhat/mercury/util/schemavalidator/AbstructJsonSchemaValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.mercury.util.schemavalidator;
+
+import org.everit.json.schema.Schema;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+public abstract class AbstructJsonSchemaValidator {
```

Comment:
There is a typo but if you remove the empty interface this can just be called `JsonSchemaValidator`